.github/workflows/ci.yml

```
@@ -253,25 +252,17 @@ jobs:
       - run: cargo install sqlx-cli
       - run: bash scripts/run_migrations.bash
 
-      - name: Build fuel-node
+      - name: Test E2E (Postgres)
         uses: actions-rs/cargo@v1
         with:
-          command: build
-          args: -p fuel-node --locked
-        env:
-          RUSTFLAGS: '-D warnings'
-
-      - name: Start test web components
-        run: |
-          CARGO_MANIFEST_DIR=$PWD/packages/fuel-indexer-tests/ target/debug/fuel-node \
-            --chain-config packages/fuel-indexer-tests/assets/test-chain-config.json \
-            --contract-bin packages/fuel-indexer-tests/contracts/fuel-indexer-test/out/debug/fuel-indexer-test.bin &
+          command: test
+          args: --features e2e,postgres e2e -- --test-threads=1
```

```suggestion
          args: --features e2e,postgres e2e --locked -- --test-threads=1
```
- I remember specifically that [`serial`](https://docs.rs/serial_test/latest/serial_test/) and `--test-threads` weren't working for me I was testing DB isolation
- Looks like some change that was made allows this now - love it